Documentation/gettingstarted/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -0,0 +1,237 @@
+.. _gs_clustermesh_aks_prep:
+
+**********************************
+AKS-to-AKS clustermesh preparation
+**********************************
+
+This is a step-by-step guide on how to install and prepare 
+AKS (Azure Kubernetes Service) clusters in BYOCNI mode to meet the requirements 
+for the clustermesh feature.
+
+In this guide we will install two AKS clusters in BYOCNI (Bring Your Own CNI) 
+mode and connect them together via clustermesh. This guide is not 
+applicable for cross-cloud clustermesh since this guide doesn't expose the node
+IPs outside of the Azure cloud.
+
+.. note::
+
+        BYOCNI requires the ``aks-preview`` CLI extension with version >=
+        0.5.55, which itself requires an ``az`` CLI version >= 2.32.0.
+
+Install cluster one
+###################
+
+1.  Create a resource group for the cluster (or set the environment variables
+    to an existing resource group).
+
+    .. code-block:: bash
+
+        export NAME="$(whoami)-$RANDOM"
+        export AZURE_RESOURCE_GROUP="${NAME}-group"
+
+        #  westus2 can be changed to any available location (`az account list-locations`)
+        az group create --name "${AZURE_RESOURCE_GROUP}" -l westus2
+
+2.  Now that we have a resource group we can create a VNet (virtual network). 
+    Creating a custom VNet is required so we can specify a unique Node, Pod, and 
+    Service CIDRs to make sure we don't overlap with other clusters.
+
+    .. note::
+        In this case we use the ``192.168.10.0/24`` range, but this can be exchanged
+        for any range except for (``169.254.0.0/16``, ``172.30.0.0/16``, 
```

Comment:
Should these parenthesis be removed? (Or the opening one be moved before the “except for”?)

Reply:
They seem unnecessary, don't know why I put them there.